src/main/java/vazkii/botania/api/brew/Brew.java

```diff
@@ -35,9 +37,19 @@
 	 * @param effects A list of effects to apply to the player when they drink it.
 	 */
 	public Brew(int color, int cost, EffectInstance... effects) {
-		this.color = color;
+		this.color = () -> color;
 		this.cost = cost;
-		this.effects = ImmutableList.copyOf(effects);
+		this.effects = () -> ImmutableList.copyOf(effects);
```

Comment:
can you make the list once and return that in the lambda instead of creating a new copy every call to the lambda?

Reply:
Does this look better?
https://github.com/Vazkii/Botania/pull/3345/files#diff-03fa907e3726ed8c32f672d932dc8752R42-R43